frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -965,8 +965,8 @@ export const dashboardLogic = kea<dashboardLogicType>([
                         // Overwrite the variable `value` from the insight
                         const resultVar: Variable = {
                             ...foundVar,
-                            value: overridenValue ?? v.value ?? foundVar.value,
-                            isNull: overridenIsNull ?? v.isNull ?? foundVar.isNull,
+                            value: overridenValue,
+                            isNull: overridenIsNull,
```

Comment:
This means we'd also show a blank state for the variable even if its used by just one insight, right? Would it be better to force all insights to use whatever the overridden dashboard value is? e.g. force a refresh of at least 1 of the 2 insights that use the variable

Reply:
Noted from call. Going to continue with the plan from initial changes in this PR. The reloading based on a variable is buggy/unclear right now. I opted to fix the UX for this instead atm